### PR TITLE
New infoblox network cmdlets

### PR DESCRIPTION
**Pull request by kylecweeks**
white space settings auto trimmed.

Added functions -
 Get-InfobloxNetwork
 Get-InfobloxNetworkExtAttributes
 Set-InfobloxNetworkExtAttributes

### CODE REVIEW

File: UMN-Infoblox.psm1 [hunk 1]
```diff
@@ -545,4 +545,150 @@ Function Set-InfobloxHostIPv6 {
...
+
+        [parameter(Mandatory)]
+        [string]$uriBase
+    )
+
+    Begin{
+        $uri = "$baseUri/$networkRef`?_return_fields=extattrs&_return_as_object=1"
+    }
+    Process{
+        $return = Invoke-RestMethod -Method Put -Uri $uri -body $body -contentType 'application/json' -WebSession $cookie
```
Comment (FISHMANPET) [inline UMN-Infoblox.psm1]: Looks like you forgot to build the body

Comment (FISHMANPET) [reply UMN-Infoblox.psm1]: Based on the help based comments I'm guessing the assumption is you're going to do something with `$ExtAttribute`, but wouldn't have the function accept a JSON formatted string, but instead a PSCustomObject or Hashtable that you'll convert to JSON yourself.  Makes it easier to pass data around if it stays as a powershell object between functions.

File: UMN-Infoblox.psm1 [hunk 2]
```diff
@@ -545,4 +545,150 @@ Function Set-InfobloxHostIPv6 {
...
+
+    .DESCRIPTION
+        Get extension properties for a network
+
+    .PARAMETER Cookie
+        Infoblox web session cookie
+
+    .PARAMETER ExtAttribute
+        The extensible attribute to set. Provide a json body in form
+        $extAttribute = @{"extattrs"=@{"$ExtAttributeKey"=@{"value"="$ExtAttributeValue"};"$extAttributeKey2"=@{"value"="$extAttributevalue2"}}} |convertto-json
```
Comment (FISHMANPET) [inline UMN-Infoblox.psm1]: As I mentioned below I think this should take a PSCustomObject or Hashtable, not a JSON formatted string, and also this  example would be easier to read either way if it looked like this:
```
$extAttribute = @{
    "extattrs" = @{
        "$ExtAttributeKey"  = @{
            "value" = "$ExtAttributeValue"
        }
        "$extAttributeKey2" = @{
            "value" = "$extAttributevalue2"
        }
    }
}
```

Comment (kylecweeks) [reply UMN-Infoblox.psm1]: Added output examples.

**Pull request merged**